cycode/cli/code_scanner.py

```diff
@@ -1099,6 +1108,35 @@ def _is_file_extension_supported(scan_type: str, filename: str) -> bool:
     return not filename.endswith(consts.SECRET_SCAN_FILE_EXTENSIONS_TO_IGNORE)
 
 
+def _handle_iac_documents(documents: List[Document]) -> List[Document]:
+    return [_handle_tflpan_document(document) for document in documents]
+
+
+def _handle_tflpan_document(document: Document) -> Document:
+    if _is_tfplan_document(document):
+        document_name = _generate_tfplan_document_name(document.path)
+        tf_content = tf_content_generator.generate_tf_content_from_tfplan(document.content)
+        return Document(document_name, tf_content, document.is_git_diff_format)
+    return document
+
+
+def _generate_tfplan_document_name(path: str) -> str:
+    document_name = change_filename_extension(path, '.tf')
+    timestamp = int(time.time())
+    return f'{timestamp}-{document_name}'
+
+
+def _is_iac(scan_type: str) -> bool:
+    return scan_type == consts.INFRA_CONFIGURATION_SCAN_TYPE
+
+
+def _is_tfplan_document(document: Document) -> bool:
+    if not document.path.endswith('.json'):
+        return False
+    tf_plan = load_json(document.content)
```

Comment:
load_json could return None. we should check that value not != None. otherwise, the code will fail below because of "TypeError: argument of type 'NoneType' is not iterable"

Reply:
I'll use .get with False fallback

Reply:
but now we have `AttributeError: 'NoneType' object has no attribute 'get'` 😢

```
tf_plan = load_json(document.content)
if tf_plan is None:
    return False

return 'resource_changes' in tf_plan
```